apim/3.x/templates/gateway/gateway-configmap.yaml

```diff
@@ -216,6 +216,10 @@ data:
             health: {{ .Values.es.lifecycle.policies.health }}
             log: {{ .Values.es.lifecycle.policies.log }}
         {{- end }}
+        settings:
```

Comment:
I think we need to add a `{{ if  .Values.es.settings}}` around the whole code you added, to be sure everything is still running fine when `number_of_shards`, `number_of_replicas` and `refresh_interval` are not explicitly defined.